DashKit/PodSDKProtocol/PodSDKLoggingShim.swift

```diff
@@ -0,0 +1,205 @@
+//
+//  PodSDKLoggingShim.swift
+//  DashKit
+//
+//  Created by Pete Schwamb on 1/21/20.
+//  Copyright © 2020 Tidepool. All rights reserved.
+//
+
+import Foundation
+import PodSDK
+import os.log
+public let log = OSLog(category: "DashPumpManager")
```

Comment:
Feels sneaky to have this global constant hiding between imports. Where is this used?

Reply:
Hah, good catch. I removed him and his sneaky accomplice above. Was unused kruft.